pkg/R/utils.R

```diff
@@ -278,6 +278,8 @@ processClosure <- function(node, oldEnv, argNames, newEnv) {
         }
       } else if (nodeChar == "$") {  # Skip the field.
         processClosure(node[[2]], oldEnv, argNames, newEnv)
+      } else if (nodeChar == "::" || nodeChar == ":::") {
```

Comment:
A slightly unrelated question -- but I was wondering what other character we could run into and the only other thing I could think of is `@`. Do we need to handle `@` somehow or does it not matter ?